launcher/minecraft/Library.cpp

```diff
@@ -172,7 +172,13 @@ QList<NetAction::Ptr> Library::getDownloads(const RuntimeContext& runtimeContext
             }
 
             if (m_repositoryURL.isEmpty()) {
-                return BuildConfig.LIBRARY_BASE + raw_storage;
+#ifndef LAUNCHER_TEST
+                QString librariesOverride = APPLICATION->settings()->get("MinecraftLibrariesURLOverride").toString();
+                if (!librariesOverride.isEmpty())
+                    return librariesOverride + raw_storage;
+                else
```

Comment:
remove the else (if the condition is true then it will exit from the previous return if not it will continue to run